### ISSUE TITLE
File not found: XML/LocalizableContent.xml

### ISSUE THREAD

**Issue by MatthieuMB** [GabuEx/my-little-investigations#35]
I tried to compile and run the game from this repository but I get the following error:

```
terminate called after throwing an instance of 'MLIException'
  what():  File not found: XML/LocalizableContent.xml
Abandon
```

It didn't took me long to realise that I need a file named XML/LocalizableContent.xml in the common.dat archive but I have no idea where I can get that file. I extracted the common.dat from the windows installer (http://www.mediafire.com/download/6bjuwci3iplr76x/MyLittleInvestigationsInstaller.exe), from the java portage of the game ( https://github.com/Steveice10/MyLittleInvestigations-Java ), from the source archive (http://www.mediafire.com/download/11h03j37asbsw0m/MyLittleInvestigationsSource.zip) but none of them have that file.

Where can I get an updated common.dat that actually work with the current commit?

**Comment by mad-mix** [comment]
You should get original game from the site, install it and update it to version 1.0.10 to get last version of common.dat file.

**Comment by mdevlamynck** [comment]
You can also grab those files from http://www.art-software.fr/files/MLI/common_data-1.0.10.tar.xz.
I use this archive to package the game for archlinux, so I try to keep up to date.

**Comment by MatthieuMB** [comment]
@mad-mix I don't have any windows or OsX machine to run the game. In fact, I already played the game with the − extremely buggy but at least working − java port and all I wanted is to compile this very repository. But I couldn't run the game because of this missing file. I tought it was kinda weird that this file wasn't downloadable anywhere, so I oppened an issue.
Running the game under windows in order to get the required file for linux is of course not a solution, all I wanted was a download link.

@mdevlamynck 
It works. Thank you very much.

So the game do complie and run with linux. Couldn't believe that wine was the only solution. (It didn't even worked)
